CMakeLists.txt

```diff
@@ -77,8 +77,8 @@ set(BUILD_SUPPORT_DIR "${CMAKE_SOURCE_DIR}/build-support")
 # CMAKE OPTIONS
 ###################################################
 
-set(TERRIER_CXXFLAGS "" CACHE STRING
```

Comment:
Some builds on Travis seems to be failing. I don't think this is your issue. It may be that those failed builds are just running on an outdated pre-built image. Talk to Wan or Pervaze.